JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeRoute.kt

```diff
@@ -222,11 +207,14 @@ private fun getHomeScreenType(
             is HomeUiState.NoPosts -> HomeScreenType.Feed
         }
     }
-    WindowSize.Expanded -> HomeScreenType.FeedWithArticleDetails
+    false -> HomeScreenType.FeedWithArticleDetails
 }
 
-private fun Modifier.homeScreenPadding(showNavRail: Boolean): Modifier = composed {
-    if (showNavRail) {
+private fun Modifier.homeScreenPadding(showTopAppBar: Boolean): Modifier = composed {
+    if (showTopAppBar) {
+        this
+            .navigationBarsPadding(bottom = false)
+    } else {
         this
             .statusBarsPadding()
```

Comment:
When the top app bar isn't shown, on dark mode the padding messes with the coloring underneath the status bar:
![device-2021-09-17-142208](https://user-images.githubusercontent.com/4217560/133855220-a9d8489b-c5db-4b8e-9d8b-2d083a032093.png)
![device-2021-09-17-142247](https://user-images.githubusercontent.com/4217560/133855222-5e7652c3-5ff7-4097-876e-cb12a6a00b09.png)